docs/en/glossary/glossary.asciidoc

```diff
@@ -323,6 +323,14 @@ role] that typically share the same hardware profile. Data tiers include the
 <<glossary-frozen-tier,frozen tier>>. See {ref}/data-tiers.html[Data tiers].
 //Source: Elasticsearch
 
+[[glossary-data-view]] data view::
+An object that enables you to select the data that you want to use in {kib} 
+and allows you to define properties of the fields. A data view can point to one or more 
```

Comment:
Since this sentence already contains "enables you to", IMO you don't need the "allows you to":
```suggestion
and define properties of the fields. A data view can point to one or more 
```
I also wonder if it should be "define the properties of the fields" or "define properties for the fields", but am not sure which is better here.